crates/augurs-prophet/src/distributions.rs

```diff
@@ -0,0 +1,51 @@
+//! Distributions required for the Prophet model.
+//!
+//! This is copied straight out of the [`statrs`] crate; full credit goes to
+//! the original authors.
+//!
+//! [`statrs`]: https://crates.io/crates/statrs
```

Comment:
Why not include the crate?

Reply:
It has a hard dependency on a couple of other crates which aren't needed so will increase build times quite a bit. I was originally worried about the final code size bloating but all the dead code should be stripped out so maybe that's not a big deal. I'll double check!

Reply:
It made literally no difference so I just added it as a dependency instead and removed the copy/pasted stuff :+1: